From 0d1b40ce22801915208ee309bab78c5d97b12999 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Maxime=20M=C3=A9r=C3=A9?= <maxime.mere@foss.st.com>
Date: Fri, 30 Aug 2024 14:48:38 +0200
Subject: [PATCH 679/954] feat(st-crypto): Change PKA compatible name
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Change compatible name of PKA to match other ST device tree

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 459bf16b6e1c0940d7e1d718375276c134a25fb2 feat(st-crypto): Change PKA compatible name]
Signed-off-by: Maxime Méré <maxime.mere@foss.st.com>
Change-Id: I2176950d06567473a23d313afd85c74d32586d14
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/401191
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 drivers/st/crypto/stm32_pka.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/st/crypto/stm32_pka.c b/drivers/st/crypto/stm32_pka.c
index 3054577420..93249370ea 100644
--- a/drivers/st/crypto/stm32_pka.c
+++ b/drivers/st/crypto/stm32_pka.c
@@ -38,7 +38,7 @@
 #define OP_NBW_FROM_SIZE(s)		OP_NBW_FROM_LEN((s) * UINT8_LEN)
 #define OP_SIZE_FROM_SIZE(s)		(OP_NBW_FROM_SIZE(s) * PKA_WORD_SIZE)
 
-#define DT_PKA_COMPAT			"st,stm32-pka64"
+#define DT_PKA_COMPAT			"st,stm32mp13-pka64"
 
 #define MAX_ECC_SIZE_LEN		640U
 #define MAX_EO_NBW			OP_NBW_FROM_LEN(MAX_ECC_SIZE_LEN)
-- 
2.53.0

